### ISSUE TITLE
NVDA does not read some list items in some dialogs of Total Commander

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3953]
**Reported by k_kolev1985 on 2014-03-05 15:03**
I've noticed that NVDA does not read some list items in some dialogs of Total Commander. Example of a list containing such list items is the "Directory menu" list in "Change directory menu" dialog. NVDA reports the items in there as list items, correctly reports their total number and the position of the currently selected one, but it does not announce the name of the currently selected list item. Similar is the behavior in the dropdown lists for choosing a drive (invoked with Alt+F1 for the left panel and with Alt+F2 for the right panel respectively) - NVDA reads only the drive letter or symbol (for items like the desktop and computer), but does not read the second part of that list item, in witch is the item label. But I don't know if the 2 issues are related or not.

Steps to reproduce it:
1. Open Total Commander.
2. Press Control+D to bring up the "Directory Hotlist" menu (alternatively, go to menu Commands -> Directory Hotlist and press Enter on it).
3. If you don't have any directories added in this menu, add a few with the "Add current dir" command in the menu.
4. Then, from the "Directory Hotlist" menu, select the "Configure" command. The "Change directory menu" dialog appears and the focus is in the "Directory menu" list.
5. Press up and down arrow to move in the list, press NVDA+UpArrow and NVDA+TAB to read the currently focused item.

Expected results:
When performing the actions in step 5, NVDA should read the name of the currently focused list item.

Actual results:
NVDA does not read the name of the currently focused item - as if it does not have one. But if you press NVDA+Control+F2 to read the object via the display model, NVDA reads the name of the item as it should.

NVDA version: next-10408,f8c857a.
OS version: Windows 7 Ultimate, SP1, 32-bit.
Total Commander version: 8.50, in english.

**Comment by leonardder** [comment]
@k-kolev1985: Can you still reproduce this in Total Commander 9.0?

**Comment by k-kolev1985** [comment]
@leonardder No, I can no longer reproduce this with Total Commander 9.0. It seams that this bug has been fixed on Total Commander's side.

However, on a 64-bit system, if you launch the 32-bit variant of Total Commander, NVDA says "selected" in front of each item in the "Directory menu" list. But I guess that is a totally different bug altogether. :)

**Comment by leonardder** [comment]
@k-kolev1985, Thanks for investigating. Is this selected bug you mention not present on Total Commander X64? If not, I suggest we'd just let it go.

**Comment by k-kolev1985** [comment]
@leonardder Yes, the "selected" bug is not present in Total Commander x64.
